tests.py

```diff
@@ -17,30 +17,19 @@
 matplotlib.use('agg')
 
 import nose
-from nose.plugins import attrib
-from matplotlib.testing.noseclasses import KnownFailure
 from matplotlib import default_test_modules
 
-plugins = [KnownFailure, attrib.Plugin]
-
-# Nose doesn't automatically instantiate all of the plugins in the
-# child processes, so we have to provide the multiprocess plugin with
-# a list.
-from nose.plugins import multiprocess
-multiprocess._instantiate_plugins = plugins
-
 
 def run(extra_args):
-    try:
-        import faulthandler
-    except ImportError:
-        pass
-    else:
-        faulthandler.enable()
+    from nose.plugins import multiprocess
+
+    matplotlib._init_tests()
 
-    if not os.path.isdir(
```

Comment:
This logic was added because mock broke 2.6 support at some point so we needed an older version on 2.6. They later reenabled this and we removed the specific version install on 2.6. All of that is irelevant when we drop 2.6 support